db/Dockerfile

```diff
@@ -1,25 +1,25 @@
-FROM postgres:9.4
+FROM postgres:9.4-alpine
+
+ENV DEFAULT_TIMEZONE UTC
 
 # Install some packages to use WAL
-RUN apt-get update \
-    && apt-get install -y \
-      build-essential \
+RUN apk add --no-cache --virtual .build-deps \
+      bash \
```

Comment:
I think we can remove this as we are aiming for smaller image.

Reply:
👍 But then we need to use `#/bin/sh` for the `entrypoint.sh` script. Should test if it also works OK with `sh`.

Reply:
not work 
`03.12.2017 1:24:49/entrypoint.sh: line 29: syntax error: unexpected "(" (expecting "fi")`

Reply:
The entrypoint script need some rewrite to work with `sh`. But I think we can achieve this.